### PR TITLE
Adding an early check to verify local package mirror is available

### PR DESCRIPTION
**Pull request by erhudy**
This is a minor fix in the Vagrantfile that will save me a lot of headaches, because I forget to restart Nginx after a computer reboot all the time.

### CODE REVIEW

File: bootstrap/vagrant_scripts/Vagrantfile [hunk 1]
```diff
@@ -35,11 +35,11 @@ config_defaults = File.join(config_path, 'bootstrap_config.sh.defaults')
 config_overrides = File.join(config_path, 'bootstrap_config.sh.overrides')
 
 File.open(config_defaults).each do |line|
-    extract_and_export_envvar(line)
+  extract_and_export_envvar(line)
 end
 if File.exist?(config_overrides)
   File.open(config_overrides).each do |line|
-      extract_and_export_envvar(line)
+    extract_and_export_envvar(line)
```
Comment (erhudy) [inline bootstrap/vagrant_scripts/Vagrantfile]: this and above are unrelated indentation fixes that were annoying me

**Comment by kamidzi** [comment]
Added a timeout equivalent to half the normal 10000 millis used elsewhere. Otherwise it can take quite some time to fail...

Exporting a bogus value for BOOTSTRAP_APT_MIRROR:

```
$ vagrant up
...
==> vm3: curl: (28) Connection timed out after 5000 milliseconds
The SSH command responded with a non-zero exit status. Vagrant
assumes that this means the command failed. The output for this command
should be in the log above. Please read the output to determine what
went wrong.
```

**Comment by erhudy** [comment]
Looks good to me.
